stochrare/tests/test_diffusion.py

```diff
@@ -212,5 +234,20 @@ def test_euler_maruyama(self):
         )
         np.testing.assert_allclose(x, np.array([1, 2.2, 9.04, 21.888]))
 
+
+class TestOrnsteinUlhenbeckProcess(unittest.TestCase):
+    def setUp(self):
+        self.oup = diffusion.OrnsteinUhlenbeck(0, 1, 1, 2, deterministic=True)
+        self.wiener = diffusion.Wiener(2, D=0.5, deterministic=True)
+        self.wiener1 = diffusion.Wiener(1, D=0.5, deterministic=True)
+
+
+    def test_update(self):
+        for wienerD in (self.wiener, self.wiener1):
+            dw = np.random.normal(size=wienerD.dimension)
+            x = np.zeros(wienerD.dimension)
+            np.testing.assert_array_equal(wienerD.update(x, 0, dw=dw), dw)
+
```

Comment:
Going forward it may be better to have 1 dedicated test class per actual class.
This `test_update` calls `OrnsteinUlhenbeckProcess.update()`, whilst `TestDynamics.test_update()` calls `DiffusionProcess.update()`. btw `TestDynamics` should then be renamed `TestDiffusionProcess`...